que.go

```diff
@@ -209,7 +209,7 @@ func execEnqueue(j *Job, txn pgxV4.Tx) error {
 		args.Status = pgtype.Present
 	}
 
-	_, err := txn.Exec(context.Background(), "que_insert_job", queue, priority, runAt, j.Type, args)
+	_, err := txn.Exec(context.Background(), sqlInsertJob, queue, priority, runAt, j.Type, args)
```

Comment:
`"que_insert_job"` is an element for a prepared statement map - where strings map to statements, specifically to `sqlInsertJob`. Since we are not using the internal connection pool that has the statement map, we will use the value directly here, instead of the key.